spec/controllers/v0/account_controls_controller_spec.rb

```diff
@@ -0,0 +1,288 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V0::AccountControlsController, type: :controller do
+  let(:service_account_access_token) { build(:service_account_access_token) }
+  let(:user_account) { create(:user_account) }
+  let(:locked) { false }
+  let(:logingov_user_verification) { create(:logingov_user_verification, user_account:, locked:) }
+  let(:idme_user_verification) { create(:idme_user_verification, user_account:, locked:) }
+  let(:dslogon_user_verification) { create(:dslogon_user_verification, user_account:, locked:) }
+  let(:mhv_user_verification) { create(:mhv_user_verification, user_account:, locked:) }
+
+  before do
+    allow_any_instance_of(V0::AccountControlsController).to receive(:authenticate_service_account).and_return(true)
```

Comment:
Don't need to do this in this PR, but would be nice to have a helper function to 'authenticate' service accounts for specs. That way we could do it the 'right' way but you don't have to implement it every spec

Reply:
Updated https://app.zenhub.com/workspaces/identity-5f5bab705a94c9001ba33734/issues/gh/department-of-veterans-affairs/va.gov-team/68600 to include this feature